src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -150,7 +142,41 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
             .get("swagger")
         }
       }
+
+    Options("/swagger/cromwell.yaml") ~>
+      swaggerUiResourceRoute ~>
+      check {
+        assertResult(StatusCodes.OK) {
+          status
+        }
+        assertResult("OK") {
+          responseAs[String]
```

Comment:
It can be anything I think. I believe the `HTTP OPTIONS` request is just to inspect the headers. I adapted jacarey's [implementation](https://github.com/broadinstitute/cromwell/commit/55464d3b4502f3bad2b8f5508a6d068a13cc5c03) that seemed to work for him, returning "OK" in the body. I'm pretty sure we could also return "" and save two bytes.

https://groups.google.com/forum/#!topic/swagger-swaggersocket/S6_I6FBjdZ8